torch/nn/modules/batchnorm.py

```diff
@@ -69,9 +79,9 @@ def forward(self, input):
         if self.training and self.track_running_stats:
             # TODO: if statement only here to tell the jit to skip emitting this when it is None
             if self.num_batches_tracked is not None:
-                self.num_batches_tracked += 1
+                self.num_batches_tracked_scalar = self.num_batches_tracked_scalar + 1
```

Comment:
Not that it matters, but you can also do += here

Reply:
`+=` actually breaks jit.script

JIT support for in-place operator on scalar is limited. `left-hand side of augmented assignment to module parameters/buffers can only be tensor types`